python/pyspark/errors/error_classes.py

```diff
@@ -15,1110 +15,16 @@
 # limitations under the License.
 #
 
-# NOTE: Automatically sort this file via
-# - cd $SPARK_HOME
-# - bin/pyspark
-# - from pyspark.errors.exceptions import _write_self; _write_self()
 import json
+from pathlib import Path
 
+THIS_DIR = Path(__file__).parent
```

Comment:
Hmmm .. I thought we should change sth in `setup.py` to include JSON as a data file.

Reply:
Actually, you are right. `MANIFEST.in` needs to be adjusted. I see the JSON file added to the `dist/` directory, but it doesn't get installed into the virtual environment I created for testing.

To find this, I had to adjust my test from `pip install -e .` to `pip install .`, so that the virtual environment gets its own copy of PySpark and does not rely on the source repo.

Fix incoming...